packages/blocks/src/page-block/edgeless/components/component-toolbar/more-button.ts

```diff
@@ -58,6 +58,7 @@ export class EdgelessMoreButton extends WithDisposable(LitElement) {
       display: block;
       color: var(--affine-text-primary-color);
       fill: currentColor;
+      font-family: var(--sl-font-sans);
```

Comment:
@RajeevDash007 though this works, but seems that the `--sl-font-sans` is the [Shoelace](https://shoelace.style/) variable that only works in the BlockSuite playground environment (we use Shoelace for debug menu here, which is not included in the exported editor), but we intend to fix this in AFFiNE. So here the correct fix should be using the `--affine` prefixed variable.